wagtail/wagtailadmin/static_src/wagtailadmin/scss/components/_streamfield.scss

```diff
@@ -6,39 +6,54 @@ ul.sequence {
 li.sequence-member {
     @include clearfix;
     position: relative;
-    border: 1px solid transparent;
+    border: solid $color-grey-4;
     border-width: 1px 0;
 
-    &:hover {
-        background-color: $color-input-focus;
-        border-color: darken($color-input-focus, 10%);
-
-        .sequence-member-inner {
-            > .struct-block > label,
-            > .char_field > label,
-            .sequence > label {
-                opacity: 1;
-            }
-        }
+    &:first-child {
+        border-top: 0;
     }
 
-    &:hover .object-help {
+    // Note this inherits .oject-help from page-editor.scss
+    .object-help {
+        display: block;
+        float: none;
+        font-style: italic;
+        margin: 0;
+        max-width: 650px;
         opacity: 1;
+        padding-left: 0;
+        position: relative;
+        width: 100%;
+
+        &::before {
+            display: none;
+        }
+        @media screen and (max-width: $breakpoint-mobile) {
+            margin: -20px 0 30px;
+            padding-left: 0;
+        }
     }
 
     .struct-block .fields {
-        @include column(10);
+        width: 100%;
         padding-left: 0;
         padding-right: 0;
     }
 
+    .struct-block label {
+        display: block;
+        float: none;
+        text-transform: uppercase;
```

Comment:
It is intentional yes. We're already changing the appearance quite dramatically by removing the float (so that the field sits below it) and the idea was to introduce consistency with this label and the parent label. That said I'm fairly agnostic about whether these should be lowercase.